editor/import/resource_importer_obj.cpp

```diff
@@ -168,18 +168,23 @@ Error ResourceImporterOBJ::import(const String &p_source_file, const String &p_s
 
 					if (face[idx].size() == 3) {
 						int norm = face[idx][2].to_int() - 1;
+						if (norm < 0)
+							norm += normals.size() + 1;
```

Comment:
[FLOWER SEEDSBASE XIII.obj.zip](https://github.com/godotengine/godot/files/961278/FLOWER.SEEDSBASE.XIII.obj.zip)

my artist create a model and export to obj.
and it contain negative index.
I have test with this model.